tests/python_test.py

```diff
@@ -478,6 +478,7 @@ def test_write_resource_invalid_align_sector(
     path = str(tmpdir.join("resources"))
     util.create_file(path, MIN_RES_SIZE)
     disks = [(path, 0)]
+    util.sanlock_is_running()
```

Comment:
Looks like leftover, I already merge this to my py3 branch:

commit f0f2a0ff50687cb06510582a845c970ab493c8f1
Author: Amit Bawer <abawer@redhat.com>
Date:   Tue May 21 14:35:40 2019 +0300

    tests: Add pytest.fixture of no_sanlock_daemon
    
    New fixture calls util.sanlock_is_running() to check if the
    pid showing in sanlock.pid file (if file exists) leads to local
    sanlock executable file to conclude if sanlock is currenlty
    running in test env. In case sanlock is running, a SunlockIsRunning
    exception will be raised from test.